tests/integration/test.ajax.node.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// This test only runs on Node.js
+var http = require('http');
+
+var adapters = ['http', 'local'];
+
+
+adapters.forEach(function (adapter) {
```

Comment:
So this isnt an intergration test, could you remove any of the `adapters` code here and put it in `/tests/components/`, the integration tests are (mostly) for core API stuff that works on all platforms, there is some node only stuff in there but that was before we have different test folders